spec/furniture/marketplace/stripe_events_controller_request_spec.rb

```diff
@@ -22,18 +26,30 @@
     double(Stripe::Charge, balance_transaction: "btx_2234")
   }
 
+  let(:square_client) {
+    double(Square::Client)
+  }
+
+  let(:square_orders_api) {
+    double(Square::OrdersApi)
+  }
```

Comment:
I decided to stop at mocking these apis. Although you could mock all the way down to the HTTP responses. Although, what would make mocking the Square connection simpler would be hiding all of the square stuff behind it's own module.

Reply:
I tend to mock at the ruby level except in system tests; and in system tests I prefer something like how we did the Stripe integration, so it winds up actually hitting a Stripe account.

Reply:
Yeah I considered this might be a stepping stone to that while I got my bearings writing system tests in our ruby codebase. What do you think about leaving as is for now while I follow-up with a full integration with a Sandbox account?

Reply:
Ah, I should be more clear; yes, this is where I would stub at the Request Spec level; and I would make assertions against the messages sent; including arguments.